### ISSUE TITLE
Loss Function Fails to Decrease

### ISSUE THREAD

**Issue by LuuuuKai** [boahK/OTMorph#6]
Hello, during the process of network training, I encountered a problem where the loss function keeps fluctuating without decreasing. Have you ever come across a similar situation in your training process?
